content/master/guides/mrd-activation-policies.md

```diff
@@ -0,0 +1,689 @@
+  - "*.rds.aws.crossplane.io"
+  - "buckets.s3.aws.crossplane.io"
+EOF
+```
+
+**Why this approach is better:**
+- **Performance**: Only activates resources you actually use
+- **Security**: Principle of least privilege for resource access
+- **Clarity**: Explicit about which resources are available
+- **Maintainability**: Easier to understand and change activation patterns
+
+## Policy basics
+
+Beyond the default policy, you can create custom activation policies that 
+specify which MRDs should activate using pattern matching:
+
+```yaml
+apiVersion: apiextensions.crossplane.io/v1alpha1
+kind: ManagedResourceActivationPolicy
+metadata:
+  name: my-policy
+spec:
+  activations:
+  - "exact-match.provider.example.com"
+  - "*.wildcard.example.com" 
+  - "prefix-*.example.com"
+```
+
+{{< hint "note" >}}
+Multiple activation policies can exist simultaneously. Their activations are 
+combined, so any MRD matched by any policy becomes active.
+{{< /hint >}}
+
+## Activation patterns
+
+### Exact matching
+
+Activate specific MRDs by their full name:
+
+```yaml
+spec:
+  activations:
+  - instances.ec2.aws.crossplane.io
+  - buckets.s3.aws.crossplane.io
+  - databases.rds.aws.crossplane.io
+```
+
+Use exact matching when:
+* You know which resources you need
+* You want fine-grained control over individual resources
+* Security policies require explicit resource approval
+
+### Wildcard matching
+
+Use wildcards to activate groups of related resources:
+
+```yaml
+spec:
+  activations:
+  - "*.rds.aws.crossplane.io"        # All RDS resources
+  - "*.storage.gcp.crossplane.io"    # All GCP storage resources  
+  - "*.compute.azure.crossplane.io"  # All Azure compute resources
+```
+
+{{< hint "tip" >}}
+Wildcard patterns only support prefix matching. The `*` must be at the 
+beginning of the pattern and match one or more DNS label components.
+{{< /hint >}}
+
+### Provider-wide activation
+
+Activate all resources from a specific provider:
+
+```yaml
+spec:
+  activations:
+  - "*.aws.crossplane.io"     # All AWS resources
+  - "*.gcp.crossplane.io"     # All GCP resources
+  - "*.azure.crossplane.io"   # All Azure resources
+```
+
+Use provider-wide activation when:
+* You're migrating from non-safe-start providers
+* Your applications use diverse resources from a single provider
+* Development environments need broad resource access
+
+## Environment-based policies
+
+### Development environment
+
+Activate minimal resources for development:
+
+```yaml
+apiVersion: apiextensions.crossplane.io/v1alpha1
+kind: ManagedResourceActivationPolicy
+metadata:
+  name: development-resources
+  namespace: development
+spec:
+  activations:
+  # Basic compute
+  - instances.ec2.aws.crossplane.io
+  - "*.compute.gcp.crossplane.io"
+  
+  # Storage
+  - buckets.s3.aws.crossplane.io
+  - "*.storage.azure.crossplane.io"
+  
+  # Databases
+  - "*.rds.aws.crossplane.io"
+  - instances.sql.gcp.crossplane.io
+```
+
+### Staging environment  
```

Comment:
Covering all of dev, staging, and prod feels a bit overkill IMO. I'd keep it at dev and prod if we give advice on this at all.